Formula/instead.rb

```diff
@@ -16,22 +17,19 @@ class Instead < Formula
   depends_on "cmake" => :build
   depends_on "pkg-config" => :build
   depends_on "gtk+3"
```

Comment:
Can this use 4?

Reply:
Doesn't look like it: https://github.com/instead-hub/instead/blob/c5fea490e037a93d6387defe87b98f11f64c9365/CMakeLists.txt#L14-L15